core/data/src/main/java/uk/gov/gchq/gaffer/data/element/Edge.java

```diff
@@ -113,6 +116,16 @@ public void putIdentifier(final IdentifierType identifierType, final Object prop
         }
     }
 
+    private void standardise() {
+        if (null != destination && null != source) {
```

Comment:
Would it be better to test to see if they're both comparable, and if so, cast them to comparable and use their natural ordering? That way if the vertices are ints or longs they will sort in a more natural order (with the source less than the destination preferably).